src/Workspaces/Core/Portable/Shared/Extensions/SemanticModelExtensions.cs

```diff
@@ -201,10 +201,14 @@ private static ISymbol MapSymbol(ISymbol symbol, ITypeSymbol type)
                 type = semanticModel.GetTypeInfo(bindableParent, cancellationToken).Type;
 
                 declaredSymbol = MapSymbol(semanticFacts.GetDeclaredSymbol(semanticModel, token, cancellationToken), type);
-                allSymbols = semanticModel.GetSymbolInfo(bindableParent, cancellationToken)
-                                              .GetBestOrAllSymbols()
-                                              .WhereAsArray(s => !s.Equals(declaredSymbol))
-                                              .SelectAsArray(s => MapSymbol(s, type));
+
+                var skipSymbolInfoLookup = declaredSymbol != null && declaredSymbol.Kind == SymbolKind.RangeVariable;
```

Comment:
:bulb: `declaredSymbol.IsKind(SymbolKind.RangeVariable)` can be used since it includes a null check.